src/components/pages/Chat/MessageAggregator.tsx

```diff
@@ -53,18 +55,24 @@ export default function MessageAggregator({ children }: Props) {
 
     const deleteRoom = useDeleteRoom()
 
+    const loadLocalMessages = useLoadLocalMessages()
```

Comment:
If we have `getLocalMessagesForRoom` then this line is gone. We import and use `getLocalMessagesForRoom`, which does not constitute a dependency for the following `useEffect`. Everyone happy.